listenbrainz/domain/spotify.py

```diff
@@ -68,7 +68,17 @@ def refresh_user_token(spotify_user):
         user (domain.spotify.Spotify): the same user with updated tokens
     """
     auth = get_spotify_oauth()
-    new_token = auth.refresh_access_token(spotify_user.refresh_token)
+
+    retries = 5
```

Comment:
either make this a config or a constant we can define at the top. Burying this in the code is no good.